src/Discovery/DiscoveryDetails/DataDownloadList/DataDownloadList.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { List, Button } from 'antd';
+import { DownloadOutlined } from '@ant-design/icons';
+import DataDownloadListItem from './Utils/DataDownloadListItem';
+import './DataDownloadList.css';
+import ProcessData from './Utils/ProcessData';
+
+const DataDownloadList = ({ sourceFieldData }: any) => {
+  const data = ProcessData(sourceFieldData);
+  if (data.length === 0) {
+    return null;
+  }
+  return (
+    <div className='discovery-modal__data-download-list'>
+      <List
+        itemLayout='horizontal'
+        dataSource={data}
+        renderItem={(item: DataDownloadListItem) => (
+          <List.Item
+            actions={[
+              <Button type='text' disabled icon={<DownloadOutlined />}>
+                Download File
```

Comment:
can we remove this place holder button for now as this effert will be done in HP-1261?

Reply:
Removed, relevant code added as a comment in associated ticket.